### DISCUSSION TITLE
colima nerdctl subcommands require `--` for option params?

### DISCUSSION THREAD

**Discussion by justin-corn** [abiosoft/colima#595] [Q&A] [answered]
Possible feature improvement, or something I have to deal with?

*Current state:*
```
colima nerdctl build . -- -t foo
```

*Desired state:*
```
colima nerdctl build . -t foo
```

## Problem Statement
I use a build tool at work that invokes the Docker CLI.  It has hooks to configure the `docker` executable and the `build` subcommand, but no hooks to specify that a `--` must intervene between positional arg params and option params.  This prevents me from using `colima` at work.

## Discussion Prompts
- am I overlooking some way to invoke without `--`?
- assuming this works like I think it does, should I open an issue, or
- is this the way it's intended to stay?

## Notes
```
$ colima version
colima version 0.4.6
git commit: 10377f3a20c2b0f7196ad5944264b69f048a3d40

runtime: containerd
arch: x86_64
client: v0.23.0
server: v1.6.8
```
Result when using no-`--`-args
```
$ colima nerdctl build . -t foo
Error: unknown shorthand flag: 't' in -t
Usage:
...
```
---
Thanks for the help!

**Comment by abiosoft** [comment, marked as answer]
It is actually supposed to be
```
colima nerdctl -- build . -t foo
```

You can run `colima nerdctl install` to add `nerdctl` script to path and you'd be able to simply run

```
nerdctl build . -t foo
```
